chapter_introduction/index_vn.md

```diff
@@ -430,7 +442,22 @@ can then simply be called the *features*,
 (or often, the *inputs*, *covariates*, or *independent variables*).
 -->
 
-*dịch đoạn phía trên*
+Có thể nói rằng bạn không thể làm khoa học dữ liệu mà không có dữ liệu.
+Chúng ta có thể mất hàng trăm trang cân nhắc chính xác những gì cấu thành dữ liệu,
+nhưng bây giờ chúng ta sẽ rẽ sang khía cạnh thực tế
+và tập trung vào các thuộc tính quan trọng cần quan tâm.
+Thông thường, chúng ta quan tâm đến một bộ *ví dụ*
+(còn được gọi là *điểm dữ liệu*, *mẫu* hoặc *trường hợp*).
+Để làm việc với dữ liệu một cách hữu ích, chúng ta thường
+cần phải đưa ra một đại diện số thích hợp.
+Mỗi *ví dụ* thường bao gồm một bộ
+thuộc tính số gọi là *đặc trưng*.
+Trong các bài toán học tập có giám sát ở trên,
+một đặc trưng đặc biệt được chọn như *mục tiêu* dự đoán,
+(đôi khi được gọi là *nhãn* hoặc *biến phụ thuộc*).
+Các đặc trưng nhất định mô hình dựa vào để đưa ra dự đoán
+có thể được gọi đơn giản là các *đặc trưng*,
+(hoặc thường là *đầu vào*, *đồng biến* hoặc *biến độc lập*).
```

Comment:
@honghanhh: bạn có thể gửi cho mình một references tiếng Việt nào mà trong đó có từ "đồng biến" hiểu như là "covariates"? Mình chưa gặp từ này bao giờ, khi nhắc tới "đồng biến" mình chỉ nghĩ tới "hàm đồng biến".

Reply:
Bạn có tham khảo ở đây :D
https://tusach.thuvienkhoahoc.com/wiki/Thu%E1%BA%ADt_ng%E1%BB%AF_ng%C3%A0nh_x%C3%A1c_su%E1%BA%A5t_th%E1%BB%91ng_k%C3%AA_v%C3%A0_h%E1%BB%8Dc_m%C3%A1y

Reply:
chắc tại comment trên của mình nói chưa rõ; khi nói references thì ý mình bao gồm: định nghĩa, diễn giải định nghĩa đó và ví dụ; nó giống như kiểu textbook (có thể online hoặc offline). Nếu không có cũng ko sao.

Reply:
Hmm, từ này không thể dịch là "đồng biến" được. Mình nghĩ google translate bị sai đoạn này. Đồng biến thường được dùng trong ngữ cảnh hàm số "đồng biến" hoặc "nghịch biến". "Đồng biến" có nghĩa là cùng hướng biến thiên.

Một từ tương tự là "covariance" được dịch là "hiệp phương sai", liệu "covariate" có nên được dịch là "hiệp biến"?

[Tham khảo](https://www.statisticshowto.datasciencecentral.com/covariate/)

> n general terms, covariates are characteristics (excluding the actual treatment) of the participants in an experiment. If you collect data on characteristics before you run an experiment, you could use that data to see how your treatment affects different groups or populations. Or, you could use that data to control for the influence of any covariate.

> Covariates may affect the outcome in a study. For example, you are running an experiment to see how corn plants tolerate drought. Level of drought is the actual “treatment”, but it isn’t the only factor that affects how plants perform: size is a known factor that affects tolerance levels, so you would run plant size as a covariate.

> A covariate can be an independent variable (i.e. of direct interest) or it can be an unwanted, confounding variable. Adding a covariate to a model can increase the accuracy of your results.

Reply:
@tiepvupsu : tùy mọi người vote thôi bác. Quan điểm của em thì là việc dịch giúp cho người đọc dễ đọc và nắm được vấn đề nhanh hơn khi đọc bản tiếng Việt so với bản gốc . Nếu 1 thuật ngữ tiếng Việt mà khi google cũng khó thấy thì chắc để kèm tiềng Anh ở bên cạnh chắc ổn hơn ( e thấy mấy bạn người Trung ngồi quanh khi đọc sách tiếng Trung thi thoảng vài trang vẫn có những keyword tiếng Anh)

Reply:
Thực ra mấy quyển sách edit tốt họ hay nói mấy từ khóa suất hiện ở trang bao nhiêu. Vậy khi tra cứu sẽ rất tiện

Reply:
từ covariates xuất hiện nhiều trong quyển này phết, lúc đầu cứ tưởng trong mỗi introduction.

Reply:
> Thực ra mấy quyển sách edit tốt họ hay nói mấy từ khóa suất hiện ở trang bao nhiêu. Vậy khi tra cứu sẽ rất tiện

"xuất hiện"

Cái đó gọi là "index". Nếu làm trực tiếp bằng latex thì chèn index tương đối dễ. Nhưng cuốn này convert từ .md ra .tex nên không làm được cái đó. Đây là một điểm trừ.

Reply:
Từ điển Lạc Việt trên MacOS dịch 'covariance' là 'tính hiệp biến' nên mình nghĩ chắc 'covariate' == 'hiệp biến' cũng được.

Reply:
Vậy từ điển Lạc Việt sai rồi. Từ "covariance" sách tiếng Việt dùng nhiều và được gọi là "hiệp phương sai".